File: C#/Binder/NullBinder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace XmlBinder.Binder
{
	/// <summary>
	/// Binder to ignore the xml tag.
	/// </summary>
	public class NullBinder : XmlBinder.Abstract.AbstractBinder
	{
		/// <summary>
		/// Create a NullBinder which will do nothing.
		/// </summary>
		public NullBinder() : base("")
		{

		}

		public override object read(object _source, string _content)
		{
			return _source;
		}

		public override string write(object _source)
		{
			return "";
		}
	}
}

File: C++/XmlBinder/Binder/nullbinder.cpp
#include "nullbinder.hpp"

NullBinder::NullBinder() : AbstractBinder("")
{

}

QObject * NullBinder::read(QObject * _source, QString _content)
{
	Q_UNUSED(_content)
	return _source;
}

QPair<QString, QObject *> NullBinder::write(QObject * _source)
{
	return QPair<QString, QObject*>("", _source);
}
